ask/prompts.py

```diff
@@ -8,11 +8,6 @@
 write it in a better way and more pythonic one
```

Comment:
I would add a github issue to replace these prompt w/ the prompt pipelines that we are using internally, would ask caleb about it, but basically you will get much better results than adding custom prompts